pulse/broker/src/main/resources/application.properties.template

```diff
@@ -45,6 +45,8 @@ queryCacheCleanupMinutes=10
 patientCacheCleanupMinutes=10
 acfCacheCleanupMinutes=30
 
+pulseOID=2.16.840.1.113883.9.224
```

Comment:
I'd not put our real OID here. I'd use it as something like `pulseOID=INSERT_THE_OID_HERE`